Classes/Service/SolrServiceProvider.php

```diff
@@ -301,14 +301,21 @@ protected function addFacetFilters(array $arguments): array
                         $queryInfo['tag'] = $this->tagForFacet($facetID);
                     }
 
-                    $this->query->createFilterQuery($queryInfo)
-                        ->setQuery($facetQuery);
+                    // If facet.missing is active and facet is selected
+                    // set solr query to exclude all known facet values
+                    if ($facetTerm == $facetInfo['config']['labelmissing']) {
```

Comment:
1. Please add the documentation / example for this parameter
2. For this parameter, please use a camelCasedVersion - `labelMissing`
3. Please use strict comparison `===`